content/css/concepts/grids/terms/grid-template-areas/grid-template-areas.md

```diff
@@ -0,0 +1,122 @@
+---
+Title: 'grid-template-areas'
+Description: 'A property that defines a grid template by referencing the names of the grid areas.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Grid'
+  - 'Layout'
+  - 'Responsive'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+Defines a grid template by referencing the names of the `grid-area` property, establishing cells in the grid and assigning them names.
+The `grid-template` property can be used as a shorthand to define the areas as well as defining the sizes of the columns and rows.
+
+## Syntax
+
+To use `grid-template-areas` it's required to assign names, using `grid-area` property on each grid item. The grid-template-areas are defined in the grid container.
```

Comment:
```suggestion
Each grid item can be assigned a name with the `grid-area` property. Then, the `grid-template-areas` utilizes these areas `.grid container`.
```

For the "Syntax" section, could we move this text below the code snippet? (also, we should mention that the `.grid-container` must have its `display` property set to `grid`.